packages/template-retail-react-app/app/components/display-price/current-price.jsx

```diff
@@ -17,42 +17,59 @@ import msg from '@salesforce/retail-react-app/app/components/display-price/messa
  * @param as - an HTML tag or component to be rendered as
  * @param isRange - show price as range or not
  * @param currency - currency to show the price in
+ * @param name - product name
  * @param extraProps - extra props to be passed into Text Component
  * @returns {JSX.Element}
  */
-const CurrentPrice = ({price, as, isRange = false, currency, ...extraProps}) => {
+const CurrentPrice = ({name, price, as, isRange = false, currency, ...extraProps}) => {
```

Comment:
Can you give this property name choice another try. It's described as "product name" .. but I feel like that might narrow the generic usability of this component. It sounds like this is something that is used specifically for accessibility, like aria label or sorts. So I'd like to see a property name that reflects that better.

Reply:
This same comment applies to the other components as well.

Reply:
Thanks for the feedback. I've changed it to `productLabelForA11y`. Let me know if that makes sense to you.